tests/js/server/replication2/replication2-replicated-state-document-store.js

```diff
@@ -609,14 +623,18 @@ const replicatedStateDocumentStoreSuiteDatabaseDeletionReplication2 = function (
     tearDown: lh.registerAgencyTestEnd,
 
     testDropDatabase: function() {
+      /*
```

Comment:
Just for clarification:

This is commented out because right now we are missing one step on delete, of collections, that should be done on CollectionGroups, and need help from supervision to figure out that the ReplicatedState does not have any participant anymore correct?

Reply:
Eventually the supervision is going to figure out that the `CollectionGroup` is empty, and clean up all the replicated states corresponding to its shard sheaves. Until that is implemented, I have modified `ClusterInfo::dropCollectionCoordinator` to lookup the replicated states corresponding to the collection shards and drop them. This test is currently commented because dropping the database goes through `ClusterInfo::dropDatabaseCoordinator`, which I have not yet adapted.
I am going to provide a PR description soon, once I'm done with that.